config/jobs/kubernetes-sigs/network-policy-api/network-policy-api-config.yaml

```diff
@@ -0,0 +1,48 @@
+presubmits:
+  kubernetes-sigs/network-policy-api:
+  - name: pull-network-policy-api-verify
+    annotations:
+      testgrid-dashboards: sig-network-policy-api
+      testgrid-tab-name: verify
+    labels:
+      preset-kind-volume-mounts: "true"
+      preset-dind-enabled: "true"
+    decorate: true
+    path_alias: sigs.k8s.io/network-policy-api
+    always_run: true
+    skip_report: false
+    spec:
+      containers:
+      - image: gcr.io/k8s-staging-test-infra/kubekins-e2e:v20221024-d0c013ee2d-master
+        command:
+          # generic runner script, handles DIND, bazelrc for caching, etc.
+          - runner.sh
+        args:
+          - make
+          - verify
+        # docker-in-docker needs privileged mode.
```

Comment:
I wonder if we need to make this privileged? With `make verify` nothing is running in docker right?

Reply:
True, we moved the lint one to a curl based instead of docker run. I'll pull it out